index.html

```diff
@@ -1099,10 +1099,10 @@ <h3>Workflows and Exchanges</h3>
       </p>
       <p>
 The exchange URL is given to the exchange client so that it can initiate the
-exchange. Note that while the exchange URL is given to the coordinator for
-provision to the exchange client, the actual exchange is performed between
-the exchange client and the workflow service; the coordinator is not involved
-after providing the exchange URL to the exchange client.
+exchange. Note that while the exchange URL is given to the coordinator to then
+provide it to the exchange client, the actual exchange is performed between the
+exchange client and the workflow service; the coordinator is not involved after
+providing the exchange URL to the exchange client.
```

Comment:
```suggestion
providing the exchange URL to the exchange client. For avoidance
of doubt, a coordinator can still use its own exchange client for any
use case that requires it to execute the exchange itself.
```

Reply:
Adjusting @dlongley's suggested addition....
```suggestion
providing the exchange URL to the exchange client. To be clear: a
coordinator can still use its own exchange client for any
use case that requires it to execute the exchange itself.
```